hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -89,6 +86,7 @@
     private transient final HoodieIndex<T> index;
     private transient final HoodieCommitArchiveLog archiveLog;
     private transient Timer.Context writeContext = null;
+    public static final SimpleDateFormat PARTITION_FORMATTER = new SimpleDateFormat("yyyy/MM/dd");
```

Comment:
Does not belong in HoodieWriteClient - HoodieWriteClient does not have a preferred partition formatter. Can you move this to the utility?